### PR TITLE
Fix/publish file list

### PR DESCRIPTION
**Pull request by milanzlicic**

### CODE REVIEW

**Comment by srdjastankovic** [comment]
@milanzlicic expose in both examples how host can be changed! Keep in demo examples default host api-demo.wolkabout.com

Lik it is in C and python: http://github.com/Wolkabout/WolkConnect-C/blob/master/examples/full_feature_set/main.chttps://integration5.wolkabout.com/#/devices/device-management/details/307

**Comment by srdjastankovic** [comment]
@milanzlicic too low info logging coverage. Only in FirmwareInstallerImpl is called loggin(INFO). Look at other wolkconnectors.

File: CMakeLists.txt [hunk 1]
```diff
@@ -69,9 +69,10 @@ file(GLOB_RECURSE TEST_SOURCE_FILES "tests/*.c" "tests/*.cpp")
 
 add_executable(${PROJECT_NAME}Tests ${TEST_SOURCE_FILES} ${TEST_HEADER_FILES})
 target_link_libraries(${PROJECT_NAME}Tests ${PROJECT_NAME} gtest_main gtest gmock_main gmock)
-target_include_directories(${PROJECT_NAME}Tests PUBLIC ${CMAKE_LIBRARY_INCLUDE_DIRECTORY})
+target_include_directories(${PROJECT_NAME}Tests SYSTEM PUBLIC ${CMAKE_LIBRARY_INCLUDE_DIRECTORY})
```
Comment (srdjastankovic) [inline CMakeLists.txt:72]: What SYSTEM do?

Comment (milanzlicic) [reply CMakeLists.txt:72]: SYSTEM ignores warnings when including headers in our code. We use this for headers of gtest and other dependencies

**Review by srdjastankovic — APPROVED** [review]

**Pull request merged**